classes/data/Authentication.class.php

```diff
@@ -60,6 +60,11 @@ class Authentication extends DBObject
             'size' => 200,
             'null' => true
         ),
+        'saml_user_identification_idp' => array(
+            'type' => 'string',
+            'size' => 200,
```

Comment:
The saml_user_identification_uid is only 170 chars because of some mysql index limitation IIRC. Though this is also quite old information and may not be relevant now.

Reply:
happy for it to be 170 if you prefer

Reply:
I added it in https://github.com/filesender/filesender/pull/2096. The database script *should* truncate it down to 170 for you there.

Reply:
Seems like a better time to do this now rather than later when/if the issue is reported with one of the storage back ends on mysql/mdb if that still exists in 2025.